apitest/testNgXmlFiles/residentApi.xml

```diff
@@ -24,13 +24,13 @@
 			<class name="io.mosip.testrig.apirig.testscripts.PostWithAutogenIdWithOtpGenerate" />
 		</classes>
```

Comment:
xml file should be unchanged. Remove the modifications done

Reply:
changes reverted